### DISCUSSION TITLE
[HELP] How can I make a user manage a Radio via API

### DISCUSSION THREAD

**Discussion by albisanti** [AzuraCast/AzuraCast#3757] [Q&A]
Hi all,
Web Radio are something new for me but I'm working with AzuraCast APIs for my customer.

What I want to understand is: **How can I create a Station and a User and make this user manage that radio via API?**

I've already managed to create the Radio with [POST] /admin/stations.
I've already created a new user with [POST] admin/users

What do I need to do to make the user manage the radio and let him get into the Radio profile?

Thank you in advance!

**Comment by YosefAdPro** [comment]
You can create new users and give them a unique API key with appropriate permissions.
